guides/web-check.md

```diff
@@ -0,0 +1,25 @@
+---
+category: Active Cyber Defence
+expires: 2019-03-15
+---
+# Web Check
+
+## The service
+
+The [Web Check Service](https://www.ncsc.gov.uk/blog-post/web-check-helping-you-secure-your-public-sector-websites) from NCSC is part of the [Active Cyber Defence](https://www.ncsc.gov.uk/blog-post/active-cyber-defence-tackling-cyber-attacks-uk) suite of services.
+
+The service scans provided URLs for a series of indicators (negative and positive technical security configurations) and reports them through a web interface, email alerts and exportable report file.
+
+Domains operated by, or on behalf of, MOJ **must** be added to Web Check under at least the central MOJ Web Check account.
+
+## How to use the service
+
+### Requirements
+
+The fully-qualified domain name or URL is required. It must be publicly accessible from the general Internet  and present as a website on HTTP (TCP/80) and/or HTTPS (TCP/443).
+
+MOJ is permitted to use the service for free as a central government organisation, but suppliers to MOJ currently are not.
+
+### Get started
+
+Contact the MOJ Cybersecurity team to be added into MOJ's subscription of the service.
```

Comment:
How does WebCheck work? If someone is added to our (Cybersecurity) subscription will it email them about all issues on all monitored services, or just the one they manage?

Reply:
@earthmoons No, WebCheck doesn't do delegate domain notification trees or anything like that. This is trying to convey that a MOJ domain must be in MOJ's central cybersecurity WebCheck account... but also they can run their own - does it not come across this way?